CHANGELOG.md

```diff
@@ -1,5 +1,17 @@
 # Changelog
 
+## [11.0.0](https://github.com/uktrade/platform-tools/compare/10.11.3...11.0.0) (2024-10-16)
+
+
+### ⚠ BREAKING CHANGES
+
+* Implement a data copy command that copies data cross-VPC ([#565](https://github.com/uktrade/platform-tools/issues/565))
```

Comment:
NOTE: This removes the previous `platform-helper database copy` command. Instructions for use of the replacement `dump` and `load` commands can be found here: https://platform.readme.trade.gov.uk/reference/copy-a-database-between-environments/